docs/using-the-form.md

```diff
@@ -0,0 +1,42 @@
+<!-- Maybe split into two routes: local and in Docker? -->
```

Comment:
I think the docker route is the higher priority route (I'm happy to write that one), should simplify the procedure